Week3/homework/step3-bonus.js

```diff
@@ -3,8 +3,8 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  const set1 = new Set(arr);
+  return [...set1];
```

Comment:
Nice! We haven't covered the spread operator, but you have put it to good use here. Alternatively, you could use the `Array.from()` method:

```js
function makeUnique(arr) {
  return Array.from(new Set(arr));
}
```